src/Sylius/Bundle/ShopBundle/Resources/views/Product/macros.html.twig

```diff
@@ -4,3 +4,23 @@
     {% endif %}
     <span> {{ sylius_calculate_price(variant)|sylius_price }} </span>
 {% endmacro %}
+
+{% macro list(products, limit = 2) %}
```

Comment:
Is there any use case we would want to have other value than `2` of the `limit` variable?